content/pages/projects.html

```diff
@@ -91,6 +91,30 @@ <h2 id="dandi">DANDI</h2>
 
   </div>
 
+  <div class="row">
+    <h2 id="datalad-registry">DataLad-Registry</h2>
+    <figure>
+      <a href="https://registry.datalad.org">
+        <img src="/theme/img/3rd/registry_logo.svg" alt="DataLad-Registry"
+             class="pull-right"
+             style="height:4.4em;padding-left:1em;margin-top:-0.3em;"/></a>
+    </figure>
+    <p>
+      <a href="https://registry.datalad.org">DataLad-Registry</a> is a service that
+      maintains up-to-date information on an expanding collection of datasets, currently
+      numbering over ten thousand. It automatically registers datasets from various
+      online sources, extracts metadata, and keeps both the datasets and their metadata
+      current. DataLad-Registry offers search functionalities, including metadata-based
+      searches, accessible through both a web interface and a RESTful API.
+    </p>
+    <ul>
+      <li>
+        <a href="https://youtu.be/_McJ1BtLsiQ?si=Pgg_4ZkYUyaksCv0">Talk</a> at
+        distribits 2024: Bringing Benefits of Centrality to Datalad
+      </li>
```

Comment:
good idea - if you could embed iframe with the talk!

Reply:
I tried `iframe`, but since it showed up too big in the page, was inconsistent with other projects, I put it in a `details` element, making it expandable. However, this setup broke the left menu and had inconsistent behavior across different browsers, as depicted below. I tried to adjust it with some customized CSS without success. At this point, I think we should just stick to a simple link to the talk which is actually more consistent with the rest of the page.

- With chrome collapsed
![Screenshot 2024-10-23 at 11 49 05 AM](https://github.com/user-attachments/assets/af171a4c-cbab-4841-b295-b6afc2f454ab)
- With chrome expanded
![Screenshot 2024-10-23 at 11 49 16 AM](https://github.com/user-attachments/assets/e3cf78d9-0b45-4068-9fb7-6a82d24f6a59)
- With safari collapsed
![Screenshot 2024-10-23 at 11 50 10 AM](https://github.com/user-attachments/assets/e70bfb22-d51e-4b50-b231-f1722d76a109)
- With safari expanded
![Screenshot 2024-10-23 at 11 50 15 AM](https://github.com/user-attachments/assets/1ab13aac-f26c-4717-8e32-c626455833a2)

If you want to try out the version depicted above, you can get it at https://github.com/candleindark/centerforopenneuroscience.org/tree/iframe until this PR is merged.

Reply:
coolio, merging/uploading iframe version

Reply:
That one has a few problems though. Particularly, it disables the menu on the left.